hack/utilities.sh

```diff
@@ -75,3 +75,12 @@ function retry() {
     ((N++))
   done
 }
+
+# Some useful colors.
+if [[ -z "${color_start-}" ]]; then
+  declare -r color_start="\033["
```

Comment:
I realize the [Google shell style guide](https://google.github.io/styleguide/shell.xml?showone=Constants_and_Environment_Variable_Names#Constants_and_Environment_Variable_Names) may not have much bearing here although I did find it quite useful, even insightful in my quest to learn more about bash. for constants it recommends ALL_CAPS names. See what you think...